src/cascade/executor/epiviz_runner.py

```diff
@@ -173,6 +204,21 @@ def has_random_effects(model):
 
 
 def main(args):
+    """
+     1. Parse arguments to the command line. The GUI calls a shell script
+        which passes the model version ID to the EpiViz Runner.
+
+     2. Construct an Execution Context, which tells EpiViz Runner where it can find
+        directories, files, and databases on the cluster.
+
+     3. Load Settings, which are the values you entered into EpiViz-AT on the web page.
+
+     4. Using the Settings and the Execution Context, fill out the data and
+        any parameters for Dismod-AT, as described in
+        :ref:`build-model-from-epiviz-settings`.
+
+     5. Put that data into a file and run Dismod-AT on that file.
+    """
     ec = make_execution_context()
     settings = load_settings(ec, args.meid, args.mvid, args.settings_file)
     add_settings_to_execution_context(ec, settings)
```

Comment:
Just saying, it seems like we have a mix of "context.<attribute> = " and "add_something(context,...)"?

Reply:
I really appreciate Natalie's observation that this variation in call semantics causes surprise. We should make the API really, really boring. I'm not up for resolving it in this PR.

Reply:
If we want one semantic, then it has to be setter functions I think.